static/locales/en-US.yaml

```diff
@@ -425,7 +425,9 @@ Change Format:
 Share:
   Share Video: Share Video
   Copy Link: Copy Link
+  Copy Link At Current Time: Copy Link At Current Time
```

Comment:
These strings shouldn't be needed with my requested changes.  Instead, let's have only one that references copying at the current time for the toggle switch.